bin/lolcommits

```diff
@@ -8,72 +8,32 @@ rescue LoadError
   require 'lolcommits'
 end
 
+require 'lolcommits/cli/fatals'
+require 'lolcommits/cli/launcher'
+require 'lolcommits/cli/process_runner'
+require 'lolcommits/cli/history_animator'
+
 include Lolcommits
+include Lolcommits::CLI
 
-require 'launchy'
 require 'choice'
-
 require 'methadone'
 include Methadone::CLILogging
```

Comment:
Not a bad idea, but since we'd probably be mixing it in to Classes anyhow, it wouldn't save us too much.  Noted for the future as part of looking at Methadone overall.